gatsby/config-options/gatsby-plugin-feed-options.tsx

```diff
@@ -197,26 +231,8 @@ const feedOptions = {
           if (introRaw.content)
             rawContent.content = [...introRaw.content, ...copyRaw.content];
 
-          const contentWithEmbeddedEntries = rawContent.content.map(
```

Comment:
I see that you refactored this part into a dedicated function. That makes it difficult to find the actual change. What did you fix aside from the refactoring? What was the issue? Can you highlight it on the counter part if this removal

Reply:
On all the other types than images, (so paragraphs, lists and so on) i did not only return the content as it was in there previous version, but checked if it has children (content.content), and then call the function recursively on those children.

In that way we get all images, even if they are very cascadenated.